resqpy/property/__init__.py

```diff
@@ -0,0 +1,20 @@
+__all__ = [
+    'PropertyCollection', 'Property', 'WellLog', 'WellIntervalProperty', 'WellIntervalPropertyCollection',
+    'WellLogCollection', 'StringLookup', 'PropertyKind', 'GridPropertyCollection',
+    'property_over_time_series_from_collection', 'property_collection_for_keyword', 'infer_property_kind',
+    'write_hdf5_and_create_xml_for_active_property', 'reformat_column_edges_to_resqml_format',
+    'reformat_column_edges_from_resqml_format', 'same_property_kind', 'selective_version_of_collection',
+    'supported_local_property_kind_list', 'supported_property_kind_list', 'supported_facet_type_list',
+    'expected_facet_type_dict', 'create_transmisibility_multiplier_property_kind'
+]
+
+from .property_common import property_collection_for_keyword, property_over_time_series_from_collection, write_hdf5_and_create_xml_for_active_property, infer_property_kind, reformat_column_edges_to_resqml_format, reformat_column_edges_from_resqml_format, selective_version_of_collection, same_property_kind, supported_property_kind_list, supported_local_property_kind_list, supported_facet_type_list, expected_facet_type_dict
```

Comment:
please split this very long line, either using a \ continuation character or simply repeating the from..import